TileFormats/Instanced3DModel/README.md

```diff
@@ -20,25 +20,39 @@ Instanced 3D Model maps well to the [ANGLE_instanced_arrays](https://www.khronos
 
 A tile is composed of a header section immediately followed by a body section.
 
+
 **Figure 1**: Instanced 3D Model layout (dashes indicate optional fields).
 
-![](figures/layout.png)
+![header layout](figures/header-layout.png)
```

Comment:
I'm open to whatever you think is best as far as naming goes. 

`span` is a little different from `scale` in quantization. In quantization since we store everything in a decode matrix `scale` is stored as `span` divided by `2^16-1`. However, here we can choose not do that and potentially improve precision which is what I am doing currently.